README.rst

```diff
@@ -8,8 +8,8 @@
 .. |AppVeyor| image:: https://ci.appveyor.com/api/projects/status/github/hyperspy/hyperspy?svg=true&branch=RELEASE_next_patch
 .. _AppVeyor: https://ci.appveyor.com/project/hyperspy/hyperspy/branch/RELEASE_next_patch
 
-.. |Coveralls| image:: https://coveralls.io/repos/github/hyperspy/hyperspy/badge.svg?branch=RELEASE_next_minor
-.. _Coveralls: https://coveralls.io/github/hyperspy/hyperspy?branch=RELEASE_next_minor
+.. |Codecov| image:: https://codecov.io/gh/hyperspy/hyperspy/branch/RELEASE_next_patch/graph/badge.svg
```

Comment:
Which version of the code do we want the badges to point to? I think the main purpose of the badges is for it to look correct for the latest release when visiting the github page and on pypi. My approach would be for it to point to the master branch and for all releases to be made off of that. Anyway this is consistent with other badges here, so pragmatically I think it's ok.

Reply:
Ignore me.... I forgot that this repo is structured unusually in terms of branch naming i.e. there is no master...